### PR TITLE
[RHPAM-2016] Update RHPAM images to use RHEL8 (UBI) base layer

### PR DESCRIPTION
**Pull request by errantepiphany**
[RHPAM-2016] Update RHPAM images to use RHEL8 (UBI) base layer
https://issues.jboss.org/browse/RHPAM-2016

Signed-off-by: David Ward <dward@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: businesscentral-monitoring/branch-overrides.yaml [hunk 1]
```diff
@@ -10,15 +10,19 @@ modules:
...
+                  ref: sprint-35
           - name: jboss-eap-modules
             git:
-                  url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: sprint-33
+                  # TODO: revert to "jboss-container-images" after this PR merges: https://github.com/jboss-container-images/jboss-eap-modules/pull/84
+                # url: https://github.com/jboss-container-images/jboss-eap-modules.git
+                  url: https://github.com/luck3y/jboss-eap-modules.git
+                  # TODO: replace with GA tag when available
+                  ref: 7.2.x-openjdk11
```
Comment (spolti) [inline businesscentral-monitoring/branch-overrides.yaml]: Wouldn't be better merge this changes on a 'snapshot' branch before push to master?

File: businesscentral-monitoring/image.yaml [hunk 2]
```diff
@@ -118,14 +121,16 @@ modules:
...
           - name: jboss.eap.cd.openshift.launch
+          - name: jboss-kie-db-drivers
+          - name: os-eap-datasource
+            version: "1.0"
           - name: jboss.eap.cd.logging
           - name: jboss.eap.config.mp-config
           - name: jboss.eap.config.jgroups
           - name: jboss.eap.config.elytron
           - name: jboss.eap.config.tracing
           - name: os-eap-probes
```
[the remark this answers is not in the document]
Comment (errantepiphany) [reply businesscentral-monitoring/image.yaml]: @spolti Yeah, good point. I'll remove the os-eap-probes line from all the images. I doesn't look like we use them in any of our templates.

[the remark this answers is not in the document]
Comment (errantepiphany) [reply businesscentral-monitoring/image.yaml]: @spolti Yeah, good point. I'll remove the os-eap-probes line from all the images. I doesn't look like we use them in any of our templates

File: businesscentral-monitoring/branch-overrides.yaml (hunk 1 above)
Comment (errantepiphany) [reply businesscentral-monitoring/branch-overrides.yaml]: @spolti Ken just said he's hoping on merging that PR later today, so hopefully I can get rid of this developer-branch dependency soon.

File: businesscentral-monitoring/image.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (spolti) [reply businesscentral-monitoring/image.yaml]: +1

File: businesscentral-monitoring/branch-overrides.yaml (hunk 1 above)
Comment (spolti) [reply businesscentral-monitoring/branch-overrides.yaml]: were the related pr merged?

**Pull request merged**